client/src/components/tabs.vue

```diff
@@ -22,7 +22,7 @@
                class="upper-right-border"
                tabindex="1"><span
                :class="tabNumber === 1 ? 'h5' : 'h5-lt'">Existing Name Request</span></v-tab>
-        <v-tabs-items class="tab-items" v-model="tabNumber">
+        <v-tabs-items class="rounded-b tab-items" v-model="tabNumber">
```

Comment:
Typo? Should be just "rounded"?

Reply:
In this case, it just needed radius for the bottom, as each upper tab was styled individually.

Reply:
Oh, I hadn't noticed that was a valid classname. All good then.